packages/menu/menu-item.md

```diff
@@ -159,3 +204,83 @@ When the `value` attribute is leveraged, the selected `<sp-menu-item>` represent
     </sp-menu-item>
 </sp-action-menu>
 ```
+
+</sp-tab-panel>
+</sp-tabs>
+
+#### Submenu
+
+An `<sp-menu-item>` can also accept content addressed to its `submenu` slot. Using the `<sp-menu>` element with this slot name the options will be surfaced in flyout menu that can be activated by hovering over the root menu item with your pointer or focusing the menu item and pressing the appropriate `ArrowRight` or `ArrowLeft` key based on text direction to move into the submenu.
+
+```html
+<sp-menu style="width: 200px;">
+    <sp-menu-item>
+        Item with submenu
+        <sp-menu slot="submenu">
+            <sp-menu-item>Additional options</sp-menu-item>
+            <sp-menu-item>Available on request</sp-menu-item>
+        </sp-menu>
+    </sp-menu-item>
+</sp-menu>
+```
+
+### Accessibility guidelines
+
+#### Include a label
+
+Use either visible slotted textContent or the `label` attribute to ensure menu items can be read by assistive technology.
```

Comment:
What does "visible slotted textContent" refer to here? Just asking for my own understanding.

Reply:
@najikahalsema how does this sound?

```
Either place visible text in the component's slot or use `label` attribute to ensure menu items can be read by assistive technology.
```